README.md

```diff
@@ -87,40 +87,34 @@ SectionGridView(
 
 | Configuration           | Type                     | Default Value           | Description                                               |
 |-------------------------|--------------------------|-------------------------|-----------------------------------------------------------|
-| `data`                  | List                     | -                       | The data to display in the grid.                           |
+| `data`                  | List                     | -                       | The data to display in the grid(for all three).                           |
```

Comment:
This is just for the Flatgrid and Simple grid

Reply:
This are still wrong, data property is only for flat and simple grid, not all three